app.js

```diff
@@ -11,52 +11,70 @@ const teamTwoSubtractButton = document.getElementById('team-two-subtract-button'
 const finishGameButton = document.getElementById('finish-game-button');
 const teamOneLabel = document.getElementById('team-one-name');
 const teamTwoLabel = document.getElementById('team-two-name');
+const teamOneInput = document.getElementById('teamNameOne');
+const teamTwoInput = document.getElementById('teamNameTwo');
 
 // create an array to hold on to the state of past games
-
+const pastGames = [];
 let name1 = '';
-let name2 =  '';
+let name2 = '';
 let score1 = 0;
 let score2 = 0;
 
-nameFormButton.addEventListener('click', (e) => {
+nameFormButton.addEventListener('click', () => {
     // get the name data from the form
-
+    name1 = teamOneInput.value;
+    name2 = teamTwoInput.value;
     // set the state to this data from the form
+    teamOneLabel.textContent = name1;
+    teamTwoLabel.textContent = name2;
 
     // reset the form values
-
+    currentGameEl.innerHTML = '';
```

Comment:
rather than currentGameEl being reset here we want to be resetting the input values to an empty string so the user doesn't have to delete the previous team names when adding new ones.